src/components/Search/SearchResults/SearchResults.tsx

```diff
@@ -23,9 +25,18 @@ const SearchResults = ({
   children,
   loadingText = 'Loading...',
   onKeyDown,
+  userInputValue = '',
 }: React.PropsWithChildren<SearchResultsProps>) => {
   const hasResults = Array.isArray(autoCompleteResults) && autoCompleteResults.length > 0;
 
+  function formatSearchLabel(label: string, searchQuery: string): React.ReactNode {
```

Comment:
this is just a personal preference thing, but I like `const formatSearchLabel = (label:string, searchQuery: string) => better`.  Also you shouldn't have to type the return type as `React.ReactNode` I think it will be inferred from the return statement below.